topics/ecology/tutorials/MetaShARK_tutorial/tutorial.md

```diff
@@ -0,0 +1,309 @@
+> You can here click on "metashark visualization" hyperlink. You can then go on the Fill In module, select 
+> the data package you created and click **Load**.
+>
+
+
+# 4] MetaShRIMPS 🦐 : Modify created metadata element
+
+To evaluate and modify metadata elements you have created, you can upload the EML xml file on the Galaxy history you created and all MetaShARK metadata templates files.
+
+<img src="./Images/15_MetaShRIMPS.png" width=100%>
+
+Open the [MetaShRIMPS interactive tool form](https://ecology.usegalaxy.eu/root?tool_id=interactive_tool_metashrimps) and select the EML xml file you just generate with MetaShARK. Clicking on Execute will launch the MetaShRIMPS R Shiny app, the message `There is an InteractiveTool result view available, waiting for view to become active... ` is displayed until the app will be ready to use. After some time, the message `There is an InteractiveTool result view available` confirm the app is deployed and you can access it clicking on **Open**. You will then have an interface looking like this:
+
+<img src="./Images/upload_1.png" width=100%>
+
+Click on the **Execute** Button, 2 new tabs called "Draft of Data Paper" and "Fair Assessment"  will appear.
+You can access all of the tool outputs by clicking on each tab (it can take a little time for your results to be displayed).
+
+# 5] Draft of Data Paper 📝
+
+<p align="justify">By clicking on the <b>"Draft of Data Paper"</b> tab, you will have access to the draft of Data Paper presented in an HTML format. You can either navigate through the Data Paper with the tabs or with the scrollbar on the right and access different elements.</p>
+
+- You can at the top of the page **download the draft** in either an _HTML format_ .... :
+
+<img src="./Images/Download_HTML.png" width=50%>
+
+- .... or an editable _docx format_ :
+
+<img src="./Images/Download_docx.png" width=50%>
+
+<hr>
+Voir comment mettre les deux photos côtés à côtes ou un truc mieux
+<hr>
+
+# 6] FAIR Quality Assessment report 📊✅
+
+[FAIR](https://www.go-fair.org/fair-principles/) stand for **Findable, Accessible, Interoperable, Reusable**. 
+<br> <br>
+
+<img src="./Images/FAIR_data_principles.jpg" width=65%>
+
+ <br>
+
+<p align="justify">These principles were to <b>improve the access and usabiliy of data</b> by the machine and to help <b>making data reusable and shareable for users</b>. It covers the whole concept of why it is necessary to produce a rich and described metadata in order to permit external users to <b>understand and reuse data</b> for their own studies.
+  
+There are several ways of computing the FAIR index, for each letter of the word is associated with a degree of FAIRitude of the data.</p>
+
+> [!NOTE]
+> The aim is to propose a metadata quality index and to see how it is possible to improve, not to punish the user 👍
+
+<hr>
+> Mettre une photo avec encadré rouge pour montrer où cliquer!!!⚠️
+<hr>
```

Comment:
should this be translated for non-bilingual audiences?

Reply:
of course, sorry....